internal/dynconfig/dynconfig.go

```diff
@@ -20,35 +20,32 @@ import (
 	"errors"
 	"time"
 
-	"d7y.io/dragonfly/v2/pkg/cache"
 	"github.com/mitchellh/mapstructure"
 )
 
-type sourceType int
+type SourceType int
 
 const (
-	// ManagerSourceType represents pulling configuration from manager
-	ManagerSourceType sourceType = 1 << iota
-
 	// LocalSourceType represents read configuration from local file
-	LocalSourceType
+	LocalSourceType SourceType = iota
+
+	// ManagerSourceType represents pulling configuration from manager
+	ManagerSourceType
 )
 
 const (
 	defaultCacheKey = "dynconfig"
 )
 
 type strategy interface {
-	Get() (interface{}, error)
-	Set(interface{})
 	Unmarshal(rawVal interface{}, opts ...DecoderConfigOption) error
 }
 
 type Dynconfig struct {
-	sourceType      sourceType
-	managerClient   managerClient
+	sourceType      SourceType
+	managerClient   ManagerClient
 	localConfigPath string
-	cache           cache.Cache
+	cachePath       string
```

Comment:
Whether it is necessary to expose the cache path of the internal implementation

Reply:
Provide default values, and users can customize.